lms/templates/credit_notifications/credit_eligibility_email.html

```diff
@@ -34,7 +34,17 @@
             </p>
 
             <p>
-              ${_(u"Congratulations! You are eligible to receive course credit for successfully completing your {platform_name} course! {link_start}Get your credit now.{link_end}").format(
+               ${_(u"Congratulations! You are eligible to receive course credit from ")}
+                % if providers:
+                     <ul>
+                       % for provider in providers:
+                        <li>
+                            ${ provider }
+                        </li>
+                       % endfor
+                     </ul>
+                % endif
+               ${_(u" for successfully completing your {platform_name} course! {link_start}Get your credit now.{link_end}").format(
```

Comment:
```
<p style="padding:5px 0">
              Congratulations! You are eligible to receive course credit for successfully completing your Your Platform Name Here course! <a href="https://ahsan-ul-haq.sandbox.edx.org/dashboard" target="_blank">Get your credit now.</a>
</p>
```
I have one email so its shows the dashboard. I will verify this again in code too.